cpp/include/nvtext/tokenize.hpp

```diff
@@ -116,7 +116,7 @@ std::unique_ptr<cudf::column> tokenize(
  * @param delimiter Strings used to separate each string into tokens.
  *                  The default of empty string will separate tokens using whitespace.
  * @param mr Device memory resource used to allocate the returned column's device memory.
- * @return New INT32 column of token counts.
+ * @return New column of cudf::size_type of token counts
```

Comment:
Some of the return type changes above removed information about the type entirely (e.g. deleting INT32). Does this function need to specially note its data type?

Reply:
Considering the awkward wording, I'm starting to prefer not including the data type in the `@return`.